packages/playground/website/.htaccess

```diff
@@ -5,7 +5,7 @@ AddEncoding x-gzip .gz
 Header unset ETag
 Header set Cache-Control "max-age=0, no-cache, no-store, must-revalidate"
 </FilesMatch>
-<FilesMatch "index\.js|blueprint-schema\.json|wp-cli.phar">
+<FilesMatch "index\.js|blueprint-schema\.json|logger.php|wp-cli.phar">
```

Comment:
Is this for Playground integrators?

Reply:
This is for localhost, but it's also required if we want to allow an integrator to send error reports.